pkgs/development/tools/build-managers/buildbot/plugins.nix

```diff
@@ -26,14 +26,14 @@ in {
   www = pythonPackages.buildPythonPackage rec {
     name = "${pname}-${version}";
     pname = "buildbot_www";
-    version = "0.9.0.post1";
+    version = "0.9.3";
 
     # NOTE: wheel is used due to buildbot circular dependency
     format = "wheel";
 
     src = fetchurl {
-      url = "https://pypi.python.org/packages/02/d0/fc56ee27a09498638a47dcc5637ee5412ab7a67bfb4b3ff47e041f3d7b66/${name}-py2-none-any.whl";
```

Comment:
Oh, interesting, the name is `buildbot_www-0.9.3-py2-none-any.whl `, so
```
fetchPypi {
  pname = "buildbot_www"; # or maybe inherit
  inherit version;
  python = "py2"; # Apparently its only compatible with Python 2.
  sha256 =  "c74951afae4520a202ec4f2265d646238bcebfb85ad0ac03d9964dcfaa79ef79";
};
```
should work.

In case any of the optional arguments has to be changed its perhaps not worth using this function instead of just using the link. What you choose is up to you.